test/e2e/deployment.go

```diff
@@ -483,8 +490,19 @@ func testRolloverDeployment(f *framework.Framework) {
 	framework.Logf("Make sure deployment %q is complete", deploymentName)
 	err = framework.WaitForDeploymentStatus(c, deployment)
 	Expect(err).NotTo(HaveOccurred())
+
+	framework.Logf("Ensure that both old replica sets have no replicas")
```

Comment:
the below can be wrapped in a func ensureReplicas(n) and reused above, but I guess it is fine